src/components/report/TermsAndConditionsDialog.jsx

```diff
@@ -1,24 +1,786 @@
 import React from 'react';
 import DialogContent from '@material-ui/core/DialogContent';
-import Text from '../Text';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
 import StandardDialog from '../StandardDialog';
 
+function H1({ children, ...rest }) {
+  return (
+    <Typography component="h1" variant="h4" gutterBottom {...rest}>
+      {children}
+    </Typography>
+  );
+}
+
+function H2({ children, ...rest }) {
+  return (
+    <Typography component="h2" variant="h5" gutterBottom {...rest}>
+      {children}
+    </Typography>
+  );
+}
+
+function H3({ children, ...rest }) {
+  return (
+    <Typography component="h3" variant="h6" gutterBottom {...rest}>
+      {children}
+    </Typography>
+  );
+}
+
+function H4({ children, ...rest }) {
+  return (
+    <Typography
+      component="h4"
+      variant="subtitle2"
+      gutterBottom
+      style={{ fontStyle: 'italic', fontWeight: 'bold' }}
+      {...rest}
+    >
+      {children}
+    </Typography>
+  );
+}
+
+function Paragraph({ children, ...rest }) {
+  return (
+    <Typography paragraph {...rest}>
+      {children}
+    </Typography>
+  );
+}
+
+function ListItem({ children, ...rest }) {
+  return (
+    <Typography component="li" {...rest}>
+      {children}
+    </Typography>
+  );
+}
+
+const useStyles = makeStyles(() => ({
```

Comment:
TIL makeStyles!.... which looks to be deprecated in v5 of material?

Also, I think @brmscheiner strongly prefers inline styling.

Reply:
yeah unless it's absolutely necessary to use makeStyles let's stick to one method everywhere (which for now is inline styles).

Reply:
Corrected in https://github.com/WildMeOrg/codex-frontend/pull/275/commits/3200d7f7d9098fd2d0fc758d0577d879396d72b9.